engine/buff/buff.cpp

```diff
@@ -620,13 +620,16 @@ buff_t::buff_t( sim_t* sim, player_t* target, player_t* source, util::string_vie
     is_fallback(),
     requires_invalidation(),
     expire_at_max_stack(),
+    ignore_mod_time_modifier( false ),
     reverse_stack_reduction( 1 ),
     current_value(),
     current_stack(),
     base_buff_duration( timespan_t::min() ),
     buff_duration_multiplier( 1.0 ),
     default_chance( 1.0 ),
     manual_chance( -1.0 ),
+    base_time_modifier_duration_multiplier( 1.0 ),
+    dynamic_time_modifier_duration_multiplier( 1.0 ),
```

Comment:
Using both "modifier" and "multiplier" in the same name is highly unusual. We don't call damage multipliers "damage modifier multiplier." I do wonder if there is a good use case for the base multiplier, these sorts of effects are always dynamic.

Reply:
> I do wonder if there is a good use case for the base multiplier

Mmmm, Time warp has an effect that would constantly set it to `1` on its' buff (it's modified by an torghast anima power). It's one of only two non-boss abilities that have time modifier stuff on them, so... 🤷 @EvanMichaels  said a split like this was normal and expected, and is why things like `recharge_multiplier` are split into base and dynamic parts in `cooldown_t`, so the base part can be set by spell data loading, while the dynamic part can be dynamically modified by transient effects.

Reply:
I suppose there's no harm in having a base mult.